account.go

```diff
@@ -224,11 +225,23 @@ func (account *account) Transact(ctx context.Context, preConditionCheck func() b
 			continue
 		}
 
+		for i := 0; i <24; i++ {
```

Comment:
I would recommend changing this to `i < 180` and use `time.Sleep(time.Second)`. This gives a better margin for error even if they attempt to use the < 2 mins gas price.

Reply:
Good point